src/interop/lib/get_mocha_tests.js

```diff
@@ -1,68 +1,20 @@
-var _ = require("lodash");
-var acorn = require("acorn");
-var walk = require("acorn/dist/walk");
-var path = require("path");
-var fs = require("fs");
+var path = require('path');
+var exec = require('sync-exec');
 
-var mochaSettings = require("./mocha_settings");
+var reporter = path.resolve(__dirname, 'test_capture.js');
+var mochaSettings = require('./mocha_settings');
+var mochaOpts = mochaSettings.mochaOpts;
+var mochaAtom = mochaSettings.mochaAtom;
 
-var sourceFolders = mochaSettings.mochaTestFolders;
+module.exports = function() {
+  var cmd = 'mocha --reporter ' + reporter + ' --opts ' + mochaOpts;
```

Comment:
I think this line might assume mocha is in PATH, we may want to go to `node_modules/.bin`.